extensions/shared/external-media/media-browser/media-item.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { Spinner } from '@wordpress/components';
+
+function MediaItem( props ) {
+	const onClick = useCallback( () => {
+		if ( props.onClick ) {
+			props.onClick( props.item );
+		}
+	}, [ props.onClick ] );
+
+	// Catch space and enter keypresses.
+	const onKeyDown = event => {
+		if ( event.which === 13 || event.which === 32 ) {
```

Comment:
`which` is deprecated and might not be supported everywhere. We should use `key` instead: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which

Reply:
Fixed in 95f6109

Reply:
See https://github.com/Automattic/jetpack/commit/95f6109c6b3ce8800143494c1db4f4d7611978d0#r39443020.